### ISSUE TITLE
Baud rate

### ISSUE THREAD

**Issue by TheFrostyboss** [ironman977/SamsungIRRemote#1]
Does the baud rate have to be 115200 for IR?

**Comment by ironman977** [comment]
No, because the IR is driven by a dedicate pin (8). Serial baud rate is set for debuging purpose and not in use, feel free to change it.
